test/controllers/GoodsLocationControllerSpec.scala

```diff
@@ -108,19 +108,14 @@ class GoodsLocationControllerSpec extends SpecBase with MockitoSugar with Nunjuc
       application.stop()
     }
 
-    "must redirect to the correct page when Border Force is submitted and Simplified Journey toggle is false" in {
+    "must redirect to the correct page for Border Force Office when valid data is submitted" in {
 
       val mockSessionRepository = mock[SessionRepository]
 
       when(mockSessionRepository.set(any())) thenReturn Future.successful(true)
 
-      val application =
-        applicationBuilder(userAnswers = Some(emptyUserAnswers))
-          .configure(Map("feature-toggles.simplifiedJourney" -> false))
-          .overrides(
-            bind[SessionRepository].toInstance(mockSessionRepository)
-          )
-          .build()
+      val userAnswers = emptyUserAnswers.set(GoodsLocationPage, GoodsLocation.values.head).success.value
+      val application = applicationBuilder(userAnswers = Some(userAnswers)).build()
```

Comment:
```suggestion
      val application = applicationBuilder(userAnswers = Some(userAnswers))
      .overrides(		
             bind[SessionRepository].toInstance(mockSessionRepository)		
           ).build()
```
I think we need to override mockSessionRepository here, please have a look